Documentation/specs/runtime-configuration-file.md

```diff
@@ -118,7 +119,10 @@ This section is copied verbatim from an identical section in the input `project.
 
 * `gcServer` - Boolean indicating if the server GC should be used (Default: _TBD_). Note: This is designed to mirror the existing [app.config](https://msdn.microsoft.com/en-us/library/ms229357.aspx) setting)
 * `gcConcurrent` - Boolean indicating if background garbage collection should be used (Default: _TBD_). Note: This is designed to mirror the existing [app.config](https://msdn.microsoft.com/en-us/library/yhwwzef8.aspx) setting).
-* `framework` - Indicates the name and version of the shared framework to use when activating the application. The presence of this section indicates that the application is a portable app designed to use a shared redistributable framework.
+* `framework` - Indicates the `name`, `version` and other properties of the shared framework to use when activating the application. The presence of this section indicates that the application is a portable app designed to use a shared redistributable framework.
+  * `autoRollForward` - When `false`, the runtime version is strictly obeyed by the host. When `autoRollForward` is not specified, or specified as `true`, then the runtime from the highest version that differs only in the `SemVer` patch field will be picked up.
+    * For example, if `version=1.0.1` and `autoRollForward` is `true`, the host would load `coreclr.dll` from `1.0.{max}` but will not load from `1.1.0` even if present. When `autoRollForward` is `false`, `coreclr.dll` will be loaded from `1.0.1` strictly.
```

Comment:
There should be no reference to Coreclr.dll. Instead, refer to the "shared framework".